pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +126,31 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	if newObj == nil {
```

Comment:
My bad; when it's creation, oldObj would be nil.

```suggestion
	if oldObj == nil {
```